tests/bootstrap.php

```diff
@@ -0,0 +1,16 @@
+<?php
```

Comment:
Wait... I'm kinda confused - I was getting autoload errors when running `vendor/bin/phing` right after I created `build.xml`, but then I think I eliminated them by adding an autoload-dev section to `composer.json`.